one_fm/api/notification.py

```diff
@@ -19,22 +19,30 @@ def get_employee_user_id(employee):
 	return frappe.get_value("Employee", {"name": employee}, "user_id")
 
 
-# This function returns the list of notification of a given user_id, within the dictionary of "name","subject".
+# This method is returning employee notification list
 @frappe.whitelist()
-def get_notification_list(user_id):
+def get_notification_list():
 	"""
 	Params:
-	User id: Employee User ID
-	Return: List of Notification
+	employee: employee ERP id 
+	Returns: notification list
 	"""
 	try:
-		notification_list = frappe.get_all("Notification Log", filters={'for_user':user_id, 'one_fm_mobile_app':1}, fields=["name","subject"])
+		notification_list = frappe.get_all("Notification Log", filters={'for_user':frappe.session.user, 'one_fm_mobile_app':1}, fields=["name","subject"])
 		if len(notification_list)>0:
-			return notification_list 
-		else:
-
-			return ('No Notification.')
+			return response("Notifications Are Listed Sucessfully", notification_list, True, 200)
```

Comment:
Please add appropriate response for the else condition (Where the length of notification list is 0).